flink-end-to-end-tests/pom.xml

```diff
@@ -35,8 +35,19 @@ under the License.
 	<name>flink-end-to-end-tests</name>
 
 	<properties>
-		<includeE2E>org.apache.flink.tests.util.categories.Dummy</includeE2E>
-		<excludeE2E></excludeE2E>
+		<includeE2E/> <!-- keep this empty -->
+		<excludeE2E/> <!-- keep this empty -->
+		<!-- functions as a global exclude; required since an empty set of included groups implicitly includes all -->
+		<includeE2E.default>org.apache.flink.tests.util.categories.Dummy</includeE2E.default>
+		<!-- default inclusions / exclusions -->
+		<e2e.include.org.apache.flink.tests.util.categories.TravisGroup1>false</e2e.include.org.apache.flink.tests.util.categories.TravisGroup1>
+		<e2e.include.org.apache.flink.tests.util.categories.TravisGroup2>false</e2e.include.org.apache.flink.tests.util.categories.TravisGroup2>
+		<e2e.include.org.apache.flink.tests.util.categories.TravisGroup3>false</e2e.include.org.apache.flink.tests.util.categories.TravisGroup3>
+		<e2e.include.org.apache.flink.tests.util.categories.TravisGroup4>false</e2e.include.org.apache.flink.tests.util.categories.TravisGroup4>
+		<e2e.include.org.apache.flink.tests.util.categories.TravisGroup5>false</e2e.include.org.apache.flink.tests.util.categories.TravisGroup5>
+		<e2e.include.org.apache.flink.tests.util.categories.TravisGroup6>false</e2e.include.org.apache.flink.tests.util.categories.TravisGroup6>
+		<e2e.exclude.org.apache.flink.tests.util.categories.Hadoop>true</e2e.exclude.org.apache.flink.tests.util.categories.Hadoop>
+		<e2e.exclude.org.apache.flink.testutils.junit.FailsOnJava11>true</e2e.exclude.org.apache.flink.testutils.junit.FailsOnJava11>
```

Comment:
Should `<e2e.include.org.apache.flink.tests.util.categories.PreCommit>false</e2e.include.org.apache.flink.tests.util.categories.PreCommit>` be added here?

Reply:
we can. technically it's not necessary, but it's a good idea to do so for clarity.

Reply:
done.